### PR TITLE
Quick fix UG

### LINKED ISSUE #593: Fix: UG table syntax issue

### PR DESCRIPTION
**Pull request by tlylt**
Fixes #593

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -472,7 +472,8 @@ A quick overview of all supported commands, their formats and examples are given
 | **CURL** | <a name="glossary-curl"></a> CURL is short for **Client URL** and is a command-line tool used in the transfer of data via different network protocols |
 | **Index** | Index in this guide refers to the position of the endpoint in the endpoint list (represented by the number beside the endpoint) |
 | **Protocol** | <a name="glossary-protocol"></a> A protocol is a system of rules that define how data is exchanged within or between systems |
-| **int**| <a name="glossary-int"></a> A primitive data type of Java that has the maximum value of (2^31)-1. The minimum value an `int` can have is -(2^31).|
+| **int** | <a name="glossary-int"></a> A primitive data type of Java that has the maximum value of (2^31)-1 and the minimum value of -(2^31) |
```
Comment (ong6) [inline docs/UserGuide.md:475]: i see you have removed a fullstop

**Review by ong6 — APPROVED** [review]
LGTM

**Review by tjtanjin — APPROVED** [review]
LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (tlylt) [reply docs/UserGuide.md:475]: 😂

**Pull request merged**